pydantic/type_adapter.py

```diff
@@ -216,7 +216,21 @@ def __init__(self, type: T, *, config: ConfigDict | None = None, _parent_depth:
             ...
 
     def __init__(self, type: Any, *, config: ConfigDict | None = None, _parent_depth: int = 2) -> None:
-        """Initializes the TypeAdapter object."""
+        """Initializes the TypeAdapter object.
+
+        Args:
+            type: The Python type associated with the `TypeAdapter`.
+            config: Configuration for the `TypeAdapter`, should be a dictionary conforming to [`ConfigDict`][pydantic.config.ConfigDict].
```

Comment:
Probably worth mentioning here that the `config` cannot be used with a model with config.
https://docs.pydantic.dev/latest/errors/usage_errors/#type-adapter-config-unused